trace/backend.go

```diff
@@ -0,0 +1,231 @@
+package trace
+
+import (
+	"bufio"
+	"context"
+	"io"
+	"net"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/stripe/veneur/protocol"
+	"github.com/stripe/veneur/ssf"
+)
+
+// DefaultBackoff defaults to 10 milliseconds of initial wait
+// time. Subsequent wait times will add this backoff to the time they
+// wait.
+const DefaultBackoff time.Duration = 20 * time.Millisecond
```

Comment:
Is there a reason you have `time.Duration` in here? I don't think it's needed (and I can think of at least one way in which it's not desirable, but it's minor).